app/assets/javascripts/toggle_name_and_email_input.js

```diff
@@ -0,0 +1,33 @@
+(function($) {
+  $.fn.toggleNameAndEmailInputs = function() {
+    this.each(function(){
+      $(this).on('click', function(){
+        var target = $($(this).data('target'));
+        var content = $($(this).data('content-target'));
+        var link = $($(this).data('link'));
+        var targetScope = $(this).data('target');
+        var store = target.detach();
+
+        content.append(link.show()).show();
+
+        if ($('#new_page').length) {
+          content.appendTo('#new_page');
+          store.appendTo('#new_page').hide();
+        }
+        else if ($('#new_mediated_page').length) {
+          content.appendTo('#new_mediated_page');
+          store.appendTo('#new_mediated_page').hide();
+        }
```

Comment:
Can we use something other than the ID of these elements here? I would be nice if, when we created a new kind of request form we didn't have to come into this plugin and ad another `else if` conditional here for it to work.